### PR TITLE
Dao with asset

### PR DESCRIPTION
**Pull request by vuvoth**
## Proposed Changes

- Create proposal with asset. 

## TODO

- [x] update change log

### CODE REVIEW

File: examples/dao/scripts/run/add_proposal_with_asset.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const { fundAccount, tryExecuteTx, ProposalType } = require("./common/common.js");
+const { accounts } = require("./common/accounts.js");
+const { mkProposalTx } = require("./common/tx-params.js");
+
+// TODO: This script should be remove after we support run algob with custom args.
+async function addProposal(runtimeEnv, deployer) {
+	// NOTE: One account only can create one proposal!
```
Comment (robert-zaremba) [inline examples/dao/scripts/run/add_proposal_with_asset.js:7]:
```suggestion
	// NOTE: There can be only one active proposal per account.
```

File: examples/dao/scripts/run/add_proposal_with_asset.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	const { _, proposer } = accounts(deployer);
+
+	// fund account
```
Comment (robert-zaremba) [inline examples/dao/scripts/run/add_proposal_with_asset.js:10]:
```suggestion
```

File: examples/dao/scripts/run/add_proposal_with_asset.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	await fundAccount(deployer, proposer);
+
+	const daoAppInfo = deployer.getApp("DAOApp");
+	const proposalLsig = deployer.getLsig("proposalLsig");
+	try {
+		await deployer.optInLsigToApp(daoAppInfo.appID, proposalLsig, {}, {});
+	} catch (e) {
+		console.log(e.message);
```
Comment (robert-zaremba) [inline examples/dao/scripts/run/add_proposal_with_asset.js:18]:
```suggestion
		console.log("can't opt in to the DAO app:", e.message);
```

**Review by robert-zaremba — APPROVED** [review]
- [x] please update the changelog.

**Comment by vuvoth** [comment]
> * [ ]  please update the changelog.

Update

**Pull request merged**